page-components/Review/ReviewSection/TimestampFeedbackCard.tsx

```diff
@@ -20,9 +22,9 @@ const FeedbackCard: React.FC<FeedbackCardProps> = ({
       </PlayButton>
       <FeedbackSection>
         <Label2 color='currentColor' marginBottom='spacing2'>
-          Time: {timeRange}
+          Time: {timeStart}-{timeEnd}
```

Comment:
How are we handling the case of spot specific feedback? i.e no range

Reply:
Yea i think i might wait on that til we make real data though? think we need to finalize the data model and whatever way we use it in the frontend will differ